org.kicad_pcb.KiCad.json

```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "org.kicad_pcb.KiCad",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "kicad",
+    "appdata-license": "GPL-2.0+ and BSD and MIT and wxWidgets",
+    "rename-desktop-file": "kicad.desktop",
+    "rename-icon": "kicad",
+    "rename-appdata-file": "kicad.appdata.xml",
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--filesystem=home"
+    ],
+    "cleanup": ["/share/doc",
+        "/share/aclocal", "/share/bakefile",
+        "/lib/wx", "/include",
+        "*.la", "*.a"],
+    "modules": [
+        {
+            "name": "libglu",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://mesa.freedesktop.org/archive/glu/glu-9.0.0.tar.bz2",
+                    "sha256": "1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12"
+                }
+            ]
+        },
+        {
+            "name": "GLEW",
```

Comment:
Oh interesting, the shared module has 2.0 here, might be worth testing and updating @TingPing @Barthalion

Reply:
Done.